tutorials/v7/draw_pave.cxx

```diff
@@ -36,18 +36,19 @@ void draw_pave()
 
    // RFrame will be automatically created as well
    auto pave = canvas->Draw<RPave>();
-   pave->AttrFill().SetColor(RColor::kBlue);
-   pave->AttrBorder().SetColor(RColor::kGreen).SetWidth(3);
-   pave->SetCornerY(-0.03_normal);
-   pave->SetHeight(0.2_normal);
+   pave->fill.color = RColor::kBlue;
+   pave->border.color = RColor::kGreen;
+   pave->border.width = 3;
+   pave->cornery = -0.03_normal;
```

Comment:
`corner_y` instead of `cornery`? Or `cornerY`, to at least stick with ROOT's camelCasing?